elektra/parallel_euler_tour_tree/euler_tour_tree.h

```diff
@@ -203,24 +196,8 @@ template <typename E> void EulerTourTreeBase<E>::Print() {
 }
 
 template <typename E>
-std::vector<std::pair<v_int, v_int>> EulerTourTreeBase<E>::Edges_() const {
-  // make a set of all edges
-  std::unordered_set<std::pair<v_int, v_int>, HashIntPairStruct> edges;
-
-  for (v_int i = 0; i < num_vertices_; i++) {
-    for (const auto &edge : vertices_[i].GetEdges()) {
-      edges.insert(edge);
-    }
-  }
-
-  // copy the edges into a vector
-  std::vector<std::pair<v_int, v_int>> edges_vec;
-  edges_vec.reserve(edges.size());
-  for (const auto &edge : edges) {
-    edges_vec.push_back(edge);
-  }
-
-  return edges_vec;
```

Comment:
I rewrote this because after switching to HdtEulerTourTree, this stuff didn't compile due to HdtEulerTourTree's HdtElement not implementing `GetEdges()`